lib/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -86,11 +88,12 @@ export const DateTimePickerToolbar: React.FC<ToolbarComponentProps> = withDefaul
         {wrapperVariant !== 'desktop' && (
           <PickerToolbar
             toolbarTitle={toolbarTitle}
-            isLandscape={false}
             penIconClassName={classes.penIcon}
             className={classes.toolbar}
             isMobileKeyboardViewOpen={isMobileKeyboardViewOpen}
             toggleMobileKeyboardView={toggleMobileKeyboardView}
+            {...other}
+            isLandscape={false}
```

Comment:
Should we allow the developers to change the orientation?

```suggestion
            isLandscape={false}
            {...other}
```

Reply:
No, DateTimePicker layout doesn't support landscape

Reply:
Make sense